Symfony/CS/Tests/Tokenizer/TokensTest.php

```diff
@@ -882,4 +882,97 @@ public function provideShortOpenTagEchoMonolithicPhpDetection()
             array("<?=' '\n?><?=' ';\n", false),
         );
     }
+
+    /**
+     * @dataProvider provideIsArray
+     * @requires PHP 5.4
+     */
+    public function testIsArray($source, array $expected)
```

Comment:
It would be great to write what is under indexes in `$expected` array.
If I would want to add some tests then I need to look in method body to saw it.
Or... use assoc array instead of numeric one.